docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+
+<img src="/static/blog/2021-survey/2.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Detractors: 10.88%, Neutrals: 32.69%, Promoters: 56.43%, Overall NPS: 46. Scale: -100 to 0 needs improvements, 0 to 30 is good, 30 to 70 is great, and 70 to 100 is excellent." />
+
+<p class="blog-description">1563 out of 1589 answered</p>
+
+### What is the main benefit you get from using MUI?
+
+<img src="/static/blog/2021-survey/3.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="This question results in 2019 - Design: 248; Time: 193; DX: 128; Components: 89; Customizability: 53; Documentation: 25; Accessibility: 5; Typescript: 5; Community support: 4; This question results in 2021 - Time: 597; Design: 407; Components: 309; DX: 306; Customizability: 181; Documentation: 57; Community support: 26; Accessibility: 24; Performance: 17" />
+
+<p class="blog-description">1422 out of 1589 answered this question in 2021</p>
+
+Here's what grew in terms of value proposition:
+
+- **Community (x2.9):** On a similar time frame, community size grew by x2.0. It looks like we're witnessing a network effect: with x2 more users, we got x3 more value from the community to the user base.
+- **Accessibility (x2.2):** Developers seem to be noticing the work we have put in, special thanks to Sebastian Silbermann [(@eps1lon)](https://github.com/eps1lon) and his continued help!
+- **Customizability (x1.6):** Apparently, you seem to be appreciating the new capabilities introduced in v5. However, it might still be far behind what we were aiming for, especially in light of the number of requests asking to better solve this problem.
+- **Components (x1.6):** The same goes for components, you seem to be enjoying the newly added components.
+  - Component quality (x1.0): It's encouraging to know you don't seem to be experience a decrease in quality. It's a risk we're taking with growing the MUI team, since it takes time for new team members to master the craft since we set the bar very high.
+- **Time (x1.4):** This is an interesting one. It doesn't feel like MUI worked on this problem specifically. Could it be because the growing pressure it the market for going faster? Or because we somehow made this value proposition more noticeable? It could also be related to the next item.
+- **Developer experience (x1.1):**
+  - Consistency (x1.75): We didn’t work on this dimension, and yet it grew. It's reasonable to assume it might be because we have more components, and then developers get to notice how consistent we make them be, highlighting this aspect.
```

Comment:
```suggestion
  - Consistency (x1.75): We didn’t work on this dimension, and yet it grew. It's reasonable to assume that because we have more components that ever before, developers have started to notice how much more consistent their work is when they use MUI.
```